include/qpid/dispatch/router_core.h

```diff
@@ -851,6 +851,30 @@ void qdr_query_free(qdr_query_t *query);
 typedef void (*qdr_manage_response_t) (void *context, const qd_amqp_error_t *status, bool more);
 void qdr_manage_handler(qdr_core_t *core, qdr_manage_response_t response_handler);
 
+typedef struct {
+    uint16_t major;
+    uint16_t minor;
+    uint16_t patch;
+    uint16_t flags;
+#define QDR_ROUTER_VERSION_SNAPSHOT 0x0100
+#define QDR_ROUTER_VERSION_RC       0x0200  // lower byte == RC #
```

Comment:
Are these used in the code anywhere?

Reply:
Nope - not yet.  Reserved.  Since we _do_ use a snapshot (dev builds), I wanted to add an additional flag to the version, which - since it's 2 bytes - aligns the version structure to a nice round 8 bytes and gives us a few extra bits to support RC numbers if we ever choose to do so in the future.